apps/rush-lib/src/logic/VersionManager.ts

```diff
@@ -105,6 +105,11 @@ export class VersionManager {
       changeManager.updateChangelog(!!shouldCommit);
     }
 
+    // Refresh rush configuration
+    this._rushConfiguration = RushConfiguration.loadFromConfigurationFile(
```

Comment:
Don't we reload the Rush config above?

Reply:
We do, but the changes between then and here cause the package.json files to be modified even further. Reloading here ensures that we have the most up-to-date projects.